client/src/main/python/slipstream/command/RunInstancesCommand.py

```diff
@@ -75,11 +75,22 @@ def _set_command_specific_options(self, parser):
                           help='Username of the user to use to contextualize the instance',
                           default='', metavar='USER')
 
+        parser.add_option('--' + NodeDecorator.NATIVE_CONTEXTUALIZATION_KEY, dest=NodeDecorator.NATIVE_CONTEXTUALIZATION_KEY,
+                          help='When SlipStream should use the native Cloud contextualization mechanisme (instead of SSH/WinRM)\nPossible values: never|always|linux only|windows only',
```

Comment:
1. Just discovered `mechanisme` here.  Please remove it as well.
2. Change `windows only` and `linux only` to be dash-separated.